vipermonkey/core/__init__.py

```diff
@@ -139,12 +140,47 @@ def __init__(self):
         # list of actions (stored as tuples by report_action)
         self.actions = []
 
+        # Track data saved in document variables.
+        self.doc_vars = {}
+
+        # Track document text.
+        self.doc_text = ""
+        
         # List of entry point functions to emulate.
         self.entry_points = ['autoopen', 'document_open', 'autoclose',
                              'document_close', 'auto_open', 'autoexec',
                              'autoexit', 'document_beforeclose', 'workbook_open',
-                             'workbook_activate', 'auto_close', 'workbook_close']
-
+                             'workbook_activate', 'auto_close', 'workbook_close',
+                             'inkpicture1_painted']
```

Comment:
is inkpicture1_painted needed here, since _Painted is handled in the callback suffixes below? (and moreover "inkpicture1" is just a default name that can be changed)